modules/stats-widget/src/stats-widget.js

```diff
@@ -47,53 +60,70 @@ export default class StatsWidget {
     this._container = null;
     this._header = null;
     this._items = {};
-    this._counter = 0;
-    this._framesPerUpdate = Math.round(Math.max(opts.framesPerUpdate || 1, 1));
-    this._formatters = {};
-    this._resetOnUpdate = {};
-
-    if (opts.formatters) {
-      for (const name in opts.formatters) {
-        let fm = opts.formatters[name];
 
-        if (typeof fm === 'string') {
-          fm = DEFAULT_FORMATTERS[fm];
-        }
-
-        this._formatters[name] = fm;
-      }
-    }
-
-    if (opts.resetOnUpdate) {
-      for (const name in opts.resetOnUpdate) {
-        this._resetOnUpdate[name] = opts.resetOnUpdate[name];
-      }
-    }
+    this._createDOM(props.container);
+  }
 
-    this._createDOM(opts.container);
+  setStats(stats) {
+    this.stats = stats;
+    this._createDOM(this._container);
```

Comment:
We will call this function many times in 3d-tiles, shouldn't we just call createDOM once in the constructor, and just reset the contents here?